README.md

```diff
@@ -0,0 +1,10 @@
+# AITH
+
+Let catch a turtle game!
+
+## Preview
+
+![Game Screenshot](https://raw.githubusercontent.com/DEKCOMSTORY/AITH/master/example.png)
+
+## Changelog
+- 2017/27/03 : Upgrade React-native-cli by #DEKCOMSTORY
```

Comment:
It would be great if the instructions for building the project is also added to `README.md` file. 💯